apps/docs/pages/guides/cli/config.tsx

```diff
@@ -39,7 +39,7 @@ export default function Config() {
           </div>
         </div>
         <div className="md:col-span-3">
-          <div className="sticky top-20 border-l">
+          <div className="sticky overflow-auto h-screen top-20 border-l">
```

Comment:
Hey @jonshung, good suggestion!

I tried your solution but it could be improved with a few things:
- we could use `max-h-*` so it can also be less high
- setting `screen` doesn't account for the top header so we can use calc with
  - 60px for the header height
  - 5rem for the `top-20` gap 
- last thing, setting a border bottom can help to give some room from the bottom of the last item

Reply:
Thanks for the improvements! I didn't thought about accounting the upper spaces. Though keeping bottom tight was on purpose since that's the current layout on prod, but having more room seems to be a more plausible choice. Thanks!